orchagent/pfcwdorch.h

```diff
@@ -75,6 +80,12 @@ class PfcWdSwOrch: public PfcWdOrch<DropHandler, ForwardHandler>
     void createEntry(const string& key, const vector<FieldValueTuple>& data);
     virtual void doTask(SelectableTimer &timer);
     //XXX Add port/queue state change event handlers
+
+    virtual bool bake() override;
```

Comment:
>override [](start = 24, length = 8)

If 'override', you can remove 'virtual'
ref: [https://stackoverflow.com/a/39932616/2514803](https://stackoverflow.com/a/39932616/2514803)